projects/plugins/jetpack/extensions/blocks/ai-image/index.js

```diff
@@ -19,21 +19,32 @@ export const settings = {
 		<Fragment>
 			<p>
 				{ __(
-					'Automatically generate an illustration for your post, powered by AI magic. We are experimenting with this feature and can tweak or remove it at any point.',
+					'Automatically generate an illustration for your post, powered by AI magic.',
 					'jetpack'
 				) }
 			</p>
+			<p>
+				{ __(
+					'We are experimenting with this feature and can tweak or remove it at any point.',
+					'jetpack'
+				) }
+			</p>
+			<ExternalLink href="https://en.forums.wordpress.com/?post_type=topic&p=3907816">
```

Comment:
I wonder if it may be worth making that a redirect link, so we can track clicks on those? Would that be interesting?

For more info: PCYsg-pY7-p2

Reply:
This is amazing!! I'm gonna use that

Reply:
Done